src/utils/serializers.ts

```diff
@@ -1,6 +1,9 @@
 import { z } from "zod";
```

Comment:
Why do we have so many parsers in a module named serializers?

I'm not sure we need this generic serializers module tbh... I'd rather move these serializers next to their respective parsers in the corresponding adapter. Adapter modules are properly interface-segmented and can hold both the parsers and serializers for each interface (bridge-api, browser, local storage, etc).

We can group together each parser/serializer pair or separate them into blocks using comments:

```
// parsers

...

// serializers

...

```

Reply:
Not sure about this movement... Can you open an issue to discuss this further in the next weeks pls?

Reply:
Sure! [Here it's](https://github.com/0xPolygonHermez/zkevm-bridge-ui/issues/176).